packages/vite/src/node/plugins/optimizedDeps.ts

```diff
@@ -70,7 +67,7 @@ export function optimizedDepsPlugin(config: ResolvedConfig): Plugin {
         // load hooks to avoid race conditions, once processing is resolved,
         // we are sure that the file has been properly save to disk
         try {
-          return loadOptimizedDep(file, depsOptimizer)
+          return await fsp.readFile(file, 'utf-8')
```

Comment:
fs.readFileSync is faster

Reply:
We don't want to block the main thread. There are a lot of other things processing in parallel, so it doesn't matter if it is a bit slower if other tasks can move on.